internal/router/middlewares/ratelim_test.go

```diff
@@ -14,126 +12,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestLimit1Addr(t *testing.T) {
```

Comment:
Definitely look close at this file.  It makes me uncomfortable to remove so many tests.  I _think_ this is ok because we simplified the rate limiting so much, but flagging this as a high risk of regression.

Reply:
yeah, makes sense